docs/get-started/using-with-map.md

```diff
@@ -44,21 +44,7 @@ If you are using react-map-gl, there are several ways to provide a token to your
 
 It is possible to use the map component without the Mapbox service, if you use another tile source (for example, if you host your own map tiles). You will need a custom Mapbox GL style that points to your own [vector tile source](https://www.mapbox.com/mapbox-gl-js/style-spec/), and pass it to `ReactMapGL` using the `mapStyle` prop. This custom style must match the schema of your tile source. 
 
-Existing free vector tile services: 
-
-- [CARTO free basemaps](https://carto.com/basemaps) for non commercial applications:
-
-
-
-| NAME | PREVIEW | LINK  |
-| -------- |-------------| -------- |
-| Positron | <img src="https://carto.com/help/images/building-maps/basemaps/positron_labels.png"  /> | https://basemaps.cartocdn.com/gl/positron-gl-style/style.json |
-| Dark Matter | <img src="https://carto.com/help/images/building-maps/basemaps/dark_labels.png"  /> | https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json |
-| Voyager | <img src="https://carto.com/help/images/building-maps/basemaps/voyager_labels.png"  /> | https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json |
-| Positron (no labels) | <img src="https://carto.com/help/images/building-maps/basemaps/positron_no_labels.png"  /> | https://basemaps.cartocdn.com/gl/positron-nolabels-gl-style/style.json |
-| Dark Matter (no labels) | <img src="https://carto.com/help/images/building-maps/basemaps/dark_no_labels.png"  /> | https://basemaps.cartocdn.com/gl/dark-matter-nolabels-gl-style/style.json |
-| Voyager (no labels) | <img src="https://carto.com/help/images/building-maps/basemaps/voyager_no_labels.png"  /> | https://basemaps.cartocdn.com/gl/voyager-nolabels-gl-style/style.json |
-
+- [CARTO free basemaps](https://carto.com/basemaps) for non commercial applications. There are 3 basemaps available [Positron, Voyager and Dark Matter](/docs/api-reference/carto/basemap)
 
 
 Ensure you follow the [Terms and Conditions](https://drive.google.com/file/d/0B3OBExqwT6KJNHp3U3VUamx6U1U/view) when using them.
```

Comment:
Is this not hosted on carto.com anywhere? It seems like linking to a webpage and not a document on Google Drive would be preferable.

Reply:
Yes, we're adding that to our website